packages/api/src/providers/fitbit.ts

```diff
@@ -510,11 +520,13 @@ export class Fitbit extends Provider implements OAuth2 {
         resp.data
       );
     } catch (error) {
-      console.log(`createSubscription for Fitbit failed. Url: ${url}, Cause: ${error}`);
+      console.log(`createSubscription for Fitbit failed. Url: ${url}, Error: ${error}`);
       capture.error(error, {
         extra: { context: `fitbit.createSubscription`, url },
       });
-      throw new Error("Failed to create a WH subscription for Fitbit", { cause: error });
+      throw new Error(`Failed to create a WH subscription for Fitbit. URL: ${url}`, {
+        cause: error,
+      });
```

Comment:
Let's avoid dynamic stuff on the error title. Also, if we don't need to wrap the error, let's just throw the original one.